### PR TITLE
added Tembo Operator GKE with Prereqs guide

### PR DESCRIPTION
**Pull request by williamong68**

### CODE REVIEW

File: docs/tembo-stacks/tembo-operator-gke.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+sidebar_position: 9
+---
```
Comment (sjmiller609) [inline docs/tembo-stacks/tembo-operator-gke.md:3]: Out of scope for this change, I think we need new top-level section called "open source", subsection "Tembo Operator" for this guide.

**Review by sjmiller609 — CHANGES_REQUESTED** [review]
I think there are issues with this and the EKS guide too, it's very unusual for a kube admin to follow instructions like this, and I think they will not like it and it makes us look amateur.

Kube admins are used to always following basically the same steps for working with any operator:

- Install something, typically with Helm
- Review the CRD's documentation
- Apply custom configurations of a CRD

So, our OSS story should work like that too.

For example, consider [CNPG's getting started guide](https://cloudnative-pg.io/documentation/1.18/quickstart/):
- part 1, have kubernetes
- part 2, Install (includes helm example)
- part 3, apply a resource

And please consider [crunchy data's operator guide](https://access.crunchydata.com/documentation/postgres-operator/4.6.2/tutorial/getting-started/):
- part 1: install operator
- part 2: create cluster

Example: 

The guide for Tembo operator should not lump in all the other dependencies, except for the CNPG operator only.


Here is what we need to do for our OSS story to be good:

- Publish to public helm repo a helm chart that installs our operator
- This helm chart has CNPG operator has a helm chart dependency
- does NOT install anything else like prometheus, traefik, calico, etc. It should only install the minimal. That other stuff is for our internal reasons an OSS user will not want that.
Guide: helm repo add... helm install ...

- comprehensive reference docs for all options on CoreDB resource
- examples and how to apply

**Comment by samay-sharma** [comment]
@sjmiller609 : Shall we close this PR?

**Comment by samay-sharma** [comment]
also cc @ianstanton we might explore this with P&G?

**Comment by ianstanton** [comment]
Let's close. We've since updated and published the helm chart so the flow is much better.

**Pull request closed without merging**